doc/changelog/08-tools/08642-vos-files.rst

```diff
@@ -0,0 +1,7 @@
+- `coqc` now provides the ability to generate compiled interfaces.
+  Use `coqc -vos foo.v` to skip all opaque proofs during the
+  compilation of `foo.v`, and output a file called `foo.vos`.
+  This feature enables working on a Coq file without the need to
+  first compile the proofs contained in its dependencies
```

Comment:
The changelog entry should contain a warning that this is an `*experimental*` feature.